code/API_definitions/device-status.yaml

```diff
@@ -183,28 +253,28 @@ paths:
               schema:
                 type: array
                 items:
-                  $ref: '#/components/schemas/EventSubscriptionInfo'
+                  $ref: "#/components/schemas/SubscriptionInfo"
         400:
-          $ref: '#/components/responses/Generic400'
+          $ref: "#/components/responses/Generic400"
         401:
-          $ref: '#/components/responses/Generic401'
+          $ref: "#/components/responses/Generic401"
         403:
-          $ref: '#/components/responses/Generic403'
+          $ref: "#/components/responses/Generic403"
         500:
-          $ref: '#/components/responses/Generic500'
+          $ref: "#/components/responses/Generic500"
         503:
-          $ref: '#/components/responses/Generic503'
-  /event-subscriptions/{eventSubscriptionId}:
+          $ref: "#/components/responses/Generic503"
+  /subscriptions/{subscriptionId}:
```

Comment:
It seems, we have missed security for this endpoint 
security:
        - threeLegged:

Reply:
Fixed